quinn-proto/src/connection/paths.rs

```diff
@@ -163,3 +177,605 @@ impl RttEstimator {
         }
     }
 }
+
+/// Implements Datagram Packetization Layer Path Maximum Transmission Unit Discovery
+///
+/// See <https://www.rfc-editor.org/rfc/rfc8899> for details
+#[derive(Clone)]
```

Comment:
This looks like it can be `Copy`.

Reply:
If we follow @Ralith's suggestion in https://github.com/quinn-rs/quinn/pull/1510#discussion_r1156223697, `MtuDiscoveryConfig` won't be `Copy` and this struct can't be `Copy` either